fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -173,8 +190,59 @@ class EqualizedOdds(ConditionalSelectionRate):
 
     short_name = "EqualizedOdds"
 
+    def __init__(self, ratio=1.0):
+        super(EqualizedOdds, self).__init__()
+        self.ratio = ratio
+
+    def load_data(self, X, y, **kwargs):
+        """Load the specified data into the object."""
+        super().load_data(X, y,
+                          event=pd.Series(y).apply(lambda y: _LABEL + "=" + str(y)),
+                          **kwargs)
+
+
+class ErrorRatio(ConditionalSelectionRate):
+    r"""Implementation of Error Ratio as a moment.
+
+    Measures the ratio in errors per attribute by overall error.
+    The 2-sided version of error ratio can be written as -
```

Comment:
nit: please remove the dash at the end of the line. It may confuse people into thinking that it's part of the math that follows